public/favicon/site.webmanifest

```diff
@@ -0,0 +1,19 @@
+{
```

Comment:
Sorry about my ignorance but, is there any particular reason of putting the manifest inside the `favicon` folder?
I'm curious because I've always placed as `manifest.json` in the `root` of the project.

Reply:
No particular reason. I create the Favicon thing with http://realfavicongenerator.net/ and it also creates the manifest.json for you.